docker-compose.full.yml

```diff
@@ -159,6 +165,23 @@ services:
     labels:
       traefik.enable: "false"
 
+  node:
+    image: ${NODE_IMAGE:-node:14.15-alpine3.12}
```

Comment:
You could use `node:lts-alpine` to make sure we are always using the latest stable version.

Reply:
Ah good point, I started with this one because it was the same version I saw locally; gonna try with lts :+1: